### PR TITLE
CD-170: XClarity (Lenovo) General/Health Monitoring

### PR DESCRIPTION
**Pull request by mariemm97**
…critpion JSON

### CODE REVIEW

File: examples/ssh/xclarity_general.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "version": "1.0.0",
+    "name": "XClarity General/Health Monitoring",
+    "description": "Retrieves general and health information from Lenovo XClarity server",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": true,
+        "sandbox_version": "1.12",
+        "others": [
+            "SSH Enabled",
+            "The keyboard_interactive option can be true or false depending on the XClarity version"
```
Comment (kgskgs) [inline examples/ssh/xclarity_general.json:12]: Which version needs what? This requirement doesn't really say anything

File: examples/ssh/xclarity_general.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "lenovo",
+        "cooling-devices",
+        "local-storage",
+        "processors",
+        "memory",
+        "system",
+        "state",
+        "restarts"
+    ],
+    "sample_period": 1800,
```
Comment (kgskgs) [inline examples/ssh/xclarity_general.json:26]: change to sample_period_s

File: examples/ssh/xclarity_general.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+ */
+function validate() {
+    executeCommand(command)
+        .then(parseValidateOutput)
+        .then(D.success)
+        .catch(checkSshError);
+}
+
+function parseValidateOutput(output) {
+    if (output.trim() !== "") {
```
Comment (GeorgiTsatsev) [inline examples/ssh/xclarity_general.js:66]: This does not validate anything

eg this output will result in a successful validation

system> Error: Command not recognized. Type 'help' to get list of supported commands. system>

File: examples/ssh/xclarity_general.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        var state = lines[i].match(/State\s+(.*)/);
+        var restarts = lines[i].match(/Restarts\s+(.*)/);
+        var coolingDevices = lines[i].match(/Cooling Devices\s+(.*)/);
+        var localStorage = lines[i].match(/Local Storage\s+(.*)/);
+        var processors = lines[i].match(/Processors\s+(.*)/);
+        var memory = lines[i].match(/Memory\s+(.*)/);
+        var system = lines[i].match(/System\s+(.*)/); 
+        if (state) variables.push(D.createVariable("state", "State", state[1], null, D.valueType.STRING))
+        if (restarts) variables.push(D.createVariable("restarts", "Restarts", restarts[1], null, D.valueType.NUMBER));
+        if (coolingDevices) variables.push(D.createVariable("coolingDevices", "Cooling Devices", coolingDevices[1], null, D.valueType.STRING));
```
Comment (GeorgiTsatsev) [inline examples/ssh/xclarity_general.js:95]: please use lower case in all uids 

eg snake_case is acceptable for this variable uid and the local_storage one

**Pull request merged**